apps/server/src/employees/employees.service.ts

```diff
@@ -22,11 +22,15 @@ export class EmployeesService {
         positionId: createEmployeeDto.positionId,
         pswdHash: await bcrypt.hash(
           createEmployeeDto.password,
-          process.env.SALT_ROUNDS || 10,
+          await bcrypt.genSalt(),
```

Comment:
does this take in a param for number of rounds?

Reply:
yes, I think I can add in 10 as a param in there